### PR TITLE
[bf16] add bf16 kernel: sigmoid & sqrt & softplus & square

### PR DESCRIPTION
**Pull request by zhangbo9674**
### PR types
New features

### PR changes
OPs

### Describe
添加`sigmoid` `sqrt` `softplus` `square` bf16 cuda kernel

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_activation_op.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import numpy as np
 from scipy.special import expit, erf
 
-from paddle.fluid.tests.unittests.op_test import OpTest, convert_float_to_uint16, skip_check_grad_ci
+from paddle.fluid.tests.unittests.op_test import OpTest, convert_float_to_uint16, skip_check_grad_ci, convert_float_to_uint16
```
Comment (zhiqiu) [inline python/paddle/fluid/tests/unittests/test_activation_op.py]: already imported

Comment (zhangbo9674) [reply python/paddle/fluid/tests/unittests/test_activation_op.py]: Done, tks!

**Review by limin2021 — APPROVED** [review]
LGTM for dropout_impl.cu.h

**Review by ZzSean — APPROVED** [review]
LGTM for op benchmark

**Review by zhiqiu — APPROVED** [review]

**Review by wanghuancoder — APPROVED** [review]
LGTM

**Pull request merged**